ada/testsuite/tests/name_resolution/formal_pkg_as_renaming_pkg/d_data.ads

```diff
@@ -0,0 +1,6 @@
+with Data_Pkg;
+
+package D_Data is
+   package Pkg_H is new Data_Pkg (Value_T => Integer);
+   package Pkg renames Pkg_H;
```

Comment:
Since you implemented resolution of several levels of renamings, should be good to test with several levels of renamings.